question12/main.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const btn = document.getElementById('js-btn');
+
+async function callApi() {
+    try {
+        const res = await fetch('https://jsondata.okiba.me/v1/json/dFQLo210903231659');
+        const json = await res.json();
+        return json.data;
+    }
+    catch(error) {
+        wrapper.textContent = "データが取得できませんでした";
+        console.error(error);
+    }
+    finally{
+        btn.remove()
+    }
+}
+async function init(){
+    const result = await callApi();
+    if (result){
+        createList(result);
+    }
```

Comment:
Must

write else block

Reply:
Thank you for the review! I added the else syntax. Please confirm.
Latest commit : https://github.com/csakyo/homework/pull/24/commits/48b06c2dadb6acb76fd5bb2780e89395ea296750